### DISCUSSION TITLE
Anyone willing to share Firewall rules

### DISCUSSION THREAD

**Discussion by presentpro** [Seji64/SniDust#53] [General]
Recently I was dealing with huge log data that overwhelmed my system. 

I now believe my server is under some sort of attack, causing the massive logs and also massive amounts of data transferred - which seems strange as it is relatively new, and I thought that having IPs on my block list would be sufficient. This is also why I have had large volumes of outbound data, which I have not yet been able to identify.

I wonder if anyone is able to share the Firewall rules they currently use to allow Snidust to run successfully, but not create a security risk. (I thought this would also make a valuable addition to the docs)

**Comment by Seji64** [comment]
Hi,
just to understand, you have entered a few allowed IPs in SniDust and your system is still being overrun?

**Reply by presentpro** [reply]
Yes, when I only have 6 specific IP addresses in the acl, all related to home networks of individuals known to me, but the system seems to have been over-run. In addition, the acl is working as intended, when I remove my own IP I am no longer able to get a response from a DNS request. I also have default domains turned off, so this is unlikely to have been related to streaming as I am changing location for a specific website only.

I fear some other vulnerability is being taken advantage of maybe because of some Firewall policies being too open.

Unfortunately logs were turned off initially, when I had over 95gb of data transferred in 1 day. Yesterday between 5-9pm I had another large volume of data, totaling an additional 8gb of data to 95.214.55.13 and 95.214.52.154, two IPs which I don't believe to be part of any expected service (at least to deliver such high data volumes).

**Reply by presentpro** [reply]
I know I can manual block these IPs, but I thought there may just be good practice firewall rules that stop it occurring in the first place (using GCP for hosting).

**Reply by Seji64** [reply]
>  In addition, the acl is working as intended, when I remove my own IP I am no longer able to get a response from a DNS request.

also a simple curl command is blocked? eg `curl YOUR_VPS_IP ` should be denied.

**Reply by presentpro** [reply]
Yes, this is denied (see response below). Although I do have an internal API running on port 5000 which will accept certain requests with a bearer token.

-----------------------------------------
HTTP/1.1 403 Forbidden
Server: nginx/1.24.0
Date: Thu, 11 Apr 2024 15:11:31 GMT
Content-Type: text/html
Content-Length: 555
Connection: keep-alive

<html>
<head><title>403 Forbidden</title></head>
<body>
<center><h1>403 Forbidden</h1></center>
<hr><center>nginx/1.24.0</center>
</body>
</html>

**Reply by Seji64** [reply]
Hm, if blocked there cant be Traffic to some ips 🧐

**Reply by presentpro** [reply]
😄 That's what I thought, but here we stand. It's not a problem specifically with the package anyway, just some sort of abuse. I'm very surprised that a new instance less than a month old would be hit by anything though.
![image](https://github.com/Seji64/SniDust/assets/166430339/8434e42a-0d4d-4cc9-b2ee-5bc4301e0c56)
